src/libstd/sys/windows/c.rs

```diff
@@ -638,6 +635,34 @@ if #[cfg(not(target_vendor = "uwp"))] {
     pub type PVECTORED_EXCEPTION_HANDLER = extern "system"
             fn(ExceptionInfo: *mut EXCEPTION_POINTERS) -> LONG;
 
+    pub const HANDLE_FLAG_INHERIT: DWORD = 0x00000001;
+
+    extern "system" {
+        pub fn SetHandleInformation(hObject: HANDLE,
+                                    dwMask: DWORD,
+                                    dwFlags: DWORD) -> BOOL;
+        pub fn AddVectoredExceptionHandler(FirstHandler: ULONG,
+                                           VectoredHandler: PVECTORED_EXCEPTION_HANDLER)
+                                           -> LPVOID;
+        pub fn CreateHardLinkW(lpSymlinkFileName: LPCWSTR,
+                               lpTargetFileName: LPCWSTR,
+                               lpSecurityAttributes: LPSECURITY_ATTRIBUTES)
+                               -> BOOL;
+    }
+}
+}
+
+// Functions forbidden when targeting the Games API partition
+cfg_if::cfg_if! {
+if #[cfg(not(target_vendor = "games"))] {
+    pub const SYMBOLIC_LINK_FLAG_DIRECTORY: DWORD = 0x1;
```

Comment:
According to the Windows SDK headers, symbolic links are only supported on desktop apps and drivers (not UWP or games):

```
#if WINAPI_FAMILY_PARTITION(WINAPI_PARTITION_DESKTOP | WINAPI_PARTITION_SYSTEM)

#if (_WIN32_WINNT >= 0x0600)

#define SYMBOLIC_LINK_FLAG_DIRECTORY                    (0x1)
#define SYMBOLIC_LINK_FLAG_ALLOW_UNPRIVILEGED_CREATE    (0x2)
```

So I think these should be also compiled out for `target_vendor = "uwp"`, but they weren't before for some reason. I can merge this with the block below if you think I should do that.